src/Mapping/DefaultQuoteStrategy.php

```diff
@@ -41,9 +44,17 @@ public function getTableName(ClassMetadata $class, AbstractPlatform $platform):
             $tableName = $class->table['schema'] . '.' . $class->table['name'];
         }
```

Comment:
Just realized this is what's cuasing the infamous dot. Instead of inlining `quoteIdentifier()`, let me try to modify this.